CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.10.2...master
+### Changed
```

Comment:
First of all, it's not changed, but fixed.
Second of all, test dependencies are not in the POM, so they're not API.
Let's remove this.

Reply:
https://github.com/atlassian/jira-actions/pull/56